Library/Formula/cig.rb

```diff
@@ -0,0 +1,17 @@
+class Cig < Formula
+  homepage "https://github.com/stevenjack/cig"
+  url "https://github.com/stevenjack/cig/archive/v0.1.1.tar.gz"
+  sha1 "4ce0822d4987e39f4a9c2d5da7234293d1404557"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["AUTO_GOPATH"] = "1"
+    system "go", "build", "-o", "cig"
+    bin.install "cig"
+  end
+
+  test do
+    system "#{bin}/cig", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!